Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -564,6 +572,13 @@ void ParsePrintMode(Option<PrintModes> option, Boogie.CommandLineParseState ps,
       options.ShowProofObligationExpressions = value;
     });
 
+    DafnyOptions.RegisterLegacyBinding(RawPointers, (options, value) => {
+      if (value && options.Get(CommonOptionBag.Target) != "rs") {
+        Console.Error.WriteLine("Error:  --raw-pointers can only be used after --target:rs or -t:rs");
```

Comment:
```suggestion
        Console.Error.WriteLine("Error:  --raw-pointers can only be used with --target:rs or -t:rs");
```

Nit obviously, but this is what made me think you were doing fancy parsing logic :)